server/src/builder.rs

```diff
@@ -355,37 +347,14 @@ async fn build_edge(
     ))
 }
 
-pub async fn build_caches_and_refreshers(
```

Comment:
This is now manually done in main so that we have a shared path for the match statement here